### PR TITLE
Added new constants and constant implementations

### PR DESCRIPTION
**Pull request by Kenneth430**

### CODE REVIEW

File: practice-project-bryce/src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -13,7 +15,110 @@
...
+
+  public static final class INTAKE {
+    public static final double AXIS_MAX_SPEED = 0.8;
+
+    public static final double DEBOUNCE_TIME_SECONDS = 0.02;
+
+    public static final double FEED_TO_SHOOTER_TIMEOUT = 0;
+    public static final double FLOOR_INTAKE_REVERSE_TIMEOUT = 0.1;
+
+    public static final double PICKUP_SPEED_PERCENT_OUTPUT = .75;
```
Comment (NolanCampbell2357) [inline practice-project-bryce/src/main/java/frc/robot/Constants.java]: Remove the constants you are not using in this project

File: practice-project-bryce/src/main/java/frc/robot/subsystems/Elevator.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    private CANSparkMax m_leftElevatorMotor;
+    private SparkPIDController m_leftPidController;
+    private RelativeEncoder m_leftEncoder;
+
+    private CANSparkMax m_rightElevatorMotor;
+
+    private int smartMotionSlot = 0;
+
+    public Elevator(){
+        m_leftElevatorMotor = new CANSparkMax(25, CANSparkLowLevel.MotorType.kBrushless);
```
Comment (NolanCampbell2357) [inline practice-project-bryce/src/main/java/frc/robot/subsystems/Elevator.java]: Can-ids should be declared in Constants.java

File: practice-project-bryce/src/main/java/frc/robot/subsystems/Elevator.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        m_leftPidController = m_leftElevatorMotor.getPIDController();
+        m_leftEncoder = m_leftElevatorMotor.getEncoder();
+
+        m_rightElevatorMotor = new CANSparkMax(31, CANSparkLowLevel.MotorType.kBrushless);
+        
+        configure();
+    }
+
+    public void configure(){
+        m_leftElevatorMotor.setInverted(false);
```
Comment (NolanCampbell2357) [inline practice-project-bryce/src/main/java/frc/robot/subsystems/Elevator.java]: All of these values you are setting into the config should be declared in Constants.java

**Review by NolanCampbell2357 — CHANGES_REQUESTED** [review]

File: practice-project-bryce/src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (Kenneth430) [reply practice-project-bryce/src/main/java/frc/robot/Constants.java]: done

File: practice-project-bryce/src/main/java/frc/robot/subsystems/Elevator.java (hunk 2 above)
Comment (Kenneth430) [reply practice-project-bryce/src/main/java/frc/robot/subsystems/Elevator.java]: done

File: practice-project-bryce/src/main/java/frc/robot/subsystems/Elevator.java (hunk 3 above)
Comment (Kenneth430) [reply practice-project-bryce/src/main/java/frc/robot/subsystems/Elevator.java]: done

**Review by NolanCampbell2357 — APPROVED** [review]

**Pull request merged**